docs/conf.py

```diff
@@ -57,7 +58,10 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = ".rst"
+source_suffix = [".rst", ".md"]
+
+# Allow using ::: in markdown files
```

Comment:
Cosmetics: the canonical spelling is Markdown, both here and in the commit log header and body.

(Ditto in the commit log body we could use "Release Please" instead of `release-please`.)